site/src/pug/includes/head.pug

```diff
@@ -14,6 +14,9 @@ link(rel="apple-touch-icon" size="128x128" href="/assets/img/logo.png")
 link(rel="stylesheet" href="/assets/css/index.css")
 link(rel="stylesheet" href="https://fonts.googleapis.com/css?family=Cabin")
 
+link(rel="stylesheet" href="https://unpkg.com/applause-button/dist/applause-button.css")
```

Comment:
imo Selectr is more important than applause so it should be loaded before applause

Reply:
I think this link & script have to be moved to the end of the body (link staying before the script tag).

> `<link rel="stylesheet">` is **allowed** in the body: https://html.spec.whatwg.org/multipage/links.html#body-ok